software/contrib/consequencer.py

```diff
@@ -72,8 +72,8 @@ def __init__(self):
         self.CvPattern_prev = 0
         self.reset_timeout = 1000
         self.maxRandomPatterns = 32  # This prevents a memory allocation error
-        self.maxCvVoltage = 9  # The maximum is 9 to maintain single digits in the voltage list
-        self.gateVoltage = 10
+        self.maxCvVoltage = europi_config["max_output_voltage"]  # The maximum is 9 to maintain single digits in the voltage list
```

Comment:
[discussion] This change seems to disagree with the comment. It seems that this script wants the max voltage to be 9. It seems like we should either leave the hardcoded value, or clamp it to 9 here.

Reply:
My vote would be to clamp it at 9. Or fix whatever issue exists in the GUI that requires 1 digit so this isn't a concern anymore

Reply:
Clamping seems to be the simplest change and more in the spirit of this PR.

Reply:
Added the call to `clamp`